Library/Homebrew/utils/analytics.rb

```diff
@@ -11,15 +11,16 @@ module Utils
   #
   # @api private
   module Analytics
+    INFLUX_BUCKET = "analytics"
+    INFLUX_TOKEN = "9eMkCRwRWS7xjPR_HbF5tBffKmnyRFSup7rq41tHZLOpnBsjVtRFd-y9R_P9OCcB3kr1ftDEzxcxTehcufy1SQ=="
+    INFLUX_HOST = "https://europe-west1-1.gcp.cloud2.influxdata.com"
+    INFLUX_ORG = "9a707721bb47fc02"
```

Comment:
Is this used anywhere?

Reply:
It's going to be used in `brew formula-analytics`. Given the previous ones are used there, it makes sense to define it here, IMO, rather than there.